Makefile

```diff
@@ -17,15 +17,34 @@ GO := go
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: clean
-clean: ## Cleanup any build binaries or packages
-	rm -rf bin vendor
+
+.PHONY: env
+env: ## Print debug information about your local environment
+	@echo git: $(shell git version)
+	@echo go: $(shell go version)
+	@echo golint: $(shell which golint)
+	@echo gofmt: $(shell which gofmt)
+	@echo staticcheck: $(shell which staticcheck)
+
+.PHONY: setup-env
+setup-env:
+	$(shell go mod download)
+	$(shell go install golang.org/x/lint/golint)
```

Comment:
shouldn't this also install staticcheck and gofmt?

Reply:
gofmt seems to already be installed and i wasn't sure how to install staticcheck because when I looked it up I couldn't find the `go get` instruction.  Maybe a followup PR to improve the env setup and tools package would be good

Reply:
:+1: